pgdrive/scene_creator/basic_utils.py

```diff
@@ -68,18 +70,58 @@ def check_lane_on_road(road_network: "RoadNetwork", lane, positive: float = 0, i
     return False
 
 
-def get_road_bounding_box(lanes):
-    line_points = get_arc_bounding_box_points(lanes[0], -1) if isinstance(lanes[0], CircularLane) \
-        else get_straight_bounding_box_points(lanes[0])
-    line_points += get_arc_bounding_box_points(lanes[-1], 1) if isinstance(lanes[-1], CircularLane) else \
-        get_straight_bounding_box_points(lanes[-1])
+def get_road_bounding_box(lanes, extra_lateral=3) -> Tuple:
+    """
+    Return (x_max, x_min, y_max, y_min) as bounding box of this road
+    :param lanes: Lanes in this road
+    :param extra_lateral: extra width in lateral direction, usually side_walk width
+    :return: x_max, x_min, y_max, y_min
+    """
+    line_points = get_arc_bounding_box_points(lanes, extra_lateral) if isinstance(lanes[0], CircularLane) \
+        else get_straight_bounding_box_points(lanes, extra_lateral)
     return get_points_bounding_box(line_points)
 
 
-def get_straight_bounding_box_points(lane):
-    start = lane.position(0.1, -lane.width / 2.0)
-    end = lane.position(lane.length - 0.1, -lane.width / 2.0)
-    return [start, end]
+def get_straight_bounding_box_points(lanes, extra_lateral) -> List:
```

Comment:
What about renaming this function to `get_straight_contour`

Reply:
ok